src/pages/form-validation/list/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useCallback, useState } from 'react';
+import {
+    AppBar,
+    Card,
+    Hidden,
+    IconButton,
+    InputProps,
+    Switch,
+    TextField,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import { Dns, Menu, Timeline } from '@material-ui/icons';
+import List from '@material-ui/core/List';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { InfoListItem } from '@pxblue/react-components';
+
+type OnChangeHandler = InputProps['onChange'];
+
+const useStyles = makeStyles((theme: Theme) => ({
+    containerWrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'start',
+        flex: '1 1 0',
+    },
+    container: {
+        maxWidth: 686,
```

Comment:
Super minor, but this `maxWidth` seems arbitrary to me despite it being the same width as the design. I commented on Ekta's angular PR with the same comment. I would at least like it to be a factor of 8 to keep in step with the rest of our design system. I ended up using 600pt in react-native. Just my two cents, do with it what you will.

Reply:
I'm cool with a factor of 8, I'll update it now.

Reply:
I went with 600 as well.